Extensions/TweenBehavior/JsExtension.js

```diff
@@ -113,20 +114,20 @@ module.exports = {
         'JsPlatform/Extensions/tween_behavior24.png',
         'JsPlatform/Extensions/tween_behavior32.png'
       )
+      .setHidden()
       .addCodeOnlyParameter('currentScene', '')
       .addParameter('identifier', _('Tween Identifier'), 'sceneTween')
       .addParameter('scenevar', _('The variable to tween'), '', false)
       .addParameter('expression', _('Initial value'), '', false)
       .addParameter('expression', _('Final value'), '', false)
-      .addParameter('expression', _('Duration'), '', false)
+      .addParameter('expression', _('Duration (in milliseconds)'), '', false)
       .addParameter('stringWithSelector', _('Easing'), easingChoices, false)
-      .setHidden()
       .getCodeExtraInformation()
-      .setIncludeFile('Extensions/TweenBehavior/shifty.js')
-      .addIncludeFile('Extensions/TweenBehavior/shifty_setup.js')
+      .setIncludeFile('Extensions/TweenBehavior/standard-easing-functions.js')
```

Comment:
By changing the included file, even if we hide this action, does it still work for previous users/projects?

Reply:
What is important is that the instruction is still linked to an implementation by `setFunctionName`. This implementation follows the same contract as before and is tested by the tests prefixed by "Legacy".
The included files changed because the implementation no longer uses Shifty.